server/controllers/userController.ts

```diff
@@ -86,7 +119,7 @@ const createUserController = (
     }
   };
 
-  return { fetchUserProfile, createUserProfile };
+  return { fetchUserProfile: getUserByUid, createUserProfile };
```

Comment:
Instead of using a variable here probably better to use the new method name `getUserByUid`.